specifications/M1/identity-trust-sts-api.yaml

```diff
@@ -0,0 +1,61 @@
+openapi: 3.0.1
+paths:
+  /token:
+    post:
+      operationId: token
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                access_token:
```

Comment:
What is the `access_token` here?

Reply:
The `access_token` refers to this [one](https://github.com/eclipse-tractusx/identity-trust/blob/main/specifications/M1/identity.protocol.base.md#6-using-the-oauth-2-client-credential-grant-to-obtain-access-tokens-from-an-sts)

Once the provider receive the `access_token` inside the SI token (from consumer/client) it needs to unwrap it and repackage it in it's own SI token.

Reply:
That's the SI token, but also the `VP Access Token` one is generated and added as claim, containing the `bearer_access_scope` if provided.

Check [here](https://github.com/eclipse-tractusx/identity-trust/blob/main/specifications/M1/identity.protocol.base.md#411-vp-access-token).

Reply:
One more question: Is the `access_token` generated by the EDC (in the context of EDC, MIW, STS etc.) and then passed to the STS for processing?

Reply:
The `access_token` will be extracted by the `EDC`  provider and then passed to the STS for being attached in the provider SI token